opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -401,6 +393,23 @@ def __init__(
         export_interval_millis: Optional[float] = None,
         export_timeout_millis: Optional[float] = None,
     ) -> None:
+        # Prioritize exporter level configuration
+        if (
+            hasattr(exporter, "_preferred_temporality")
```

Comment:
Why is the `hasattr` check needed? Looks like it is always present in the base `MetricExporter` class

Reply:
Some exporter implementations might not call `super()`.

Reply:
Why wouldn't they call super class constructor? It would be easy to break things if they forget to call it, for example this line
https://github.com/open-telemetry/opentelemetry-python/blob/db633ecc051d7e5822c57f6c19932efaef5b0a28/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py#L288

would cause `AttributeError: 'MetricReader' object has no attribute '_collect'`

Reply:
Okay. If that is an assumption we are making (users who override OT base classes will always call `super()` that's fine with me.